react/components/organisms/toggle-buttons/toggle-buttons.js

```diff
@@ -0,0 +1,98 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, View, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { IdentifiableMixin } from "../../../util";
+
+import { ToggleButton } from "../../molecules";
+
+export class ToggleButtons extends mix(PureComponent).with(IdentifiableMixin) {

```

Comment:
We've used `ButtonGroup` in the past (https://ripe-twitch-ui-git-master-platforme.vercel.app/storybook?path=/story/components-molecules-button-group--button-group) and I feel it sounds more clear. WDYT?

Reply:
Also renamed toggle-button to button-toggle, I forget in the previous PR